src/redux/products/products.reducer.js

```diff
@@ -193,6 +199,32 @@ const productsReducer = (state = initialState, action = {}) => {
       ...state,
       productLoading: action.payload
     };
+  case SET_RATE:
```

Comment:
I think it is better to move all the redux data related to the comments to the separate reducer `Comments`.

Reply:
moved